nashville-software-school/yak-25
# Create basic Navbar using bootstrap

Steps to test:
* ```git fetch --all```
* ```got checkout ask-nav```
* npm install (there are new packages)
* npm start 

What to Expect:
* Navbar should show up with bootstrap styling
* You should be able to click on Logo and my profile button and navigate
